pages/program/abstracts/student-program.md

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
There will need to be more changes in this PR to actually make it show up in the page tree:
- https://github.com/USRSE/usrse24/blob/main/_data/navigation.yml
- https://github.com/USRSE/usrse24/blob/main/_data/menus/program.yml

Reply:
Done in two most recent commits